spec.html

```diff
@@ -35771,6 +36365,42 @@ <h1>
           </emu-alg>
         </emu-clause>
 
+        <emu-clause id="sec-maybesimplecasefolding" type="abstract operation">
+          <h1>
+            MaybeSimpleCaseFolding (
+              _A_: a CharSet,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>If _UnicodeSets_ is *false* or _IgnoreCase_ is *false*, it returns _A_. Otherwise, it uses the <a href="https://www.unicode.org/reports/tr44/#Simple_Case_Folding">Simple Case Folding</a> (<dfn>scf</dfn>) definitions in the file <a href="https://unicode.org/Public/UCD/latest/ucd/CaseFolding.txt"><code>CaseFolding.txt</code></a> of the Unicode Character Database (each of which maps a single code point to another single code point) to map each CharSetElement of _A_ character-by-character into a canonical form and returns the resulting CharSet.</dd>
```

Comment:
UnicodeSets and IgnoreCase are not explicit parameters to this AO, and should be.

Reply:
We need to follow the upstream changes in #2716, no doubt (it’s on the TODO list). I think these wouldn’t be explicit parameters but rather internal slots that we read (e.g. `_rer_.[[UnicodeSets]]`)

Reply:
ah yes, that's exactly right :-)